Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -58,14 +58,14 @@ AlternateWebMPlayerEnabled:
   type: bool
   humanReadableName: "Alternate WebM Player"
   humanReadableDescription: "Enable Alternate WebM Player"
+  getter: isAlternateWebMPlayerEnabled
+  webcoreBinding: DeprecatedGlobalSettings
```

Comment:
Please don't add more to 'DeprecatedGlobalSettings'. The name is meant to make it clear this is something that should not be used.

Instead, all new settings should live on WebCore::Settings, and if lower levels of the stack need them, delegation or other means can be used to make them available.

Reply:
How do you recommend exposing these settings to a static context?

Reply:
The alternatives I have though of have been to use `RuntimeEnabledFeatures` which I understand is also deprecated, or to pass a `WebCore::Settings` object into the static context and into the GPU Process every time the page updates (or I can just pass the individual boolean around instead of the whole object).

The first alternative is very similar to using `DeprecatedGlobalSettings` and the second alternative has issues since the GPU Process is shared amongst different pages (which one gets priority?).

Reply:
Good points, I'll discuss this further with the rest of the media team.